fhir-client-config/pom.xml

```diff
@@ -71,5 +71,11 @@
 			<artifactId>mockito-core</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-collections4</artifactId>
```

Comment:
This is a new dependency for all of our projects. Is there any need to introduce Collections v4 or can it be just the v1 API like the rest of our projects?

Reply:
Whoops, let my IDE add the dependency. Just rerunning the tests on the other dependency now, but I think that should be fine.